.github/workflows/docs.yml

```
@@ -0,0 +1,53 @@
+name: Publish Rust Docs
+
+on:
+    #TODO we really only want to publish on pushes to master.
+    # But for now I'm testing the ci
+    pull_request:
+        branches:
+            - master
+
+jobs:
+    deploy-docs:
+      name: Deploy docs
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v1
+        # This is how Frontier got the toolchain installed
+        # - name: Rust Setup
+        #   run: |
+        #     scripts/init.sh
+        #     cargo --version
+        #     rustc --version
+        #     cargo +$WASM_BUILD_TOOLCHAIN --version
+        #     rustc +$WASM_BUILD_TOOLCHAIN --version
+        #   env:
+        #     WASM_BUILD_TOOLCHAIN: nightly-2020-08-29
+        - uses: actions-rs/toolchain@v1
+          with:
+            target: wasm32-unknown-unknown
+            # Toolchain is autodetected from `rust-toolchain` file
+            # https://github.com/actions-rs/toolchain#the-toolchain-file
+            #toolchain: ${{ env.WASM_BUILD_TOOLCHAIN }}
+            default: true
+
+        - name: Build rustdocs
+          uses: actions-rs/cargo@v1
+          with:
+            command: doc
+            args: --all --no-deps=
```

the  `--no-deps=` seems weird. is it expected like that ?

---

```suggestion
            args: --all --no-deps
```

You're right. I don't know how that got in there, but it isn't like that in frontier https://github.com/paritytech/frontier/blob/8d54307c84d95bff1300e2803296464bbbe65b64/.github/workflows/docs.yml#L36